lib/sass.dart

```diff
@@ -15,12 +16,12 @@ import 'src/visitor/serialize.dart';
 /// If [color] is `true`, this will use terminal colors in warnings.
 ///
 /// Throws a [SassException] if conversion fails.
-String render(String path, {bool color: false}) {
+Future<String> render(String path, {bool color: false}) async {
```

Comment:
This function should take the `SyncPackageResolver`, not `_PerformVisitor`. Rendering needs to be synchronous in order to be compatible with the node-sass API.

Also, there should be a command-line `--packages` flag that allows the user to manually specify a package spec.